### PR TITLE
Fix: Jira Issue Tracker fixes

### PR DESCRIPTION
**Pull request by andrei-near**
Jira module version bump to latest supported with python3.7
Jira Token auth instead of user/password
Jira issue tracker example configuration in docs
Fixes in Jira Issue tracker code
Additional AppEngine logs to make debugging easier
Cosmetic change in testcase details page to show Jira issue key instead of ID

### CODE REVIEW

File: src/appengine/libs/issue_management/issue_filer.py [hunk 1]
```diff
@@ -334,6 +334,8 @@ def file_issue(testcase,
       is_security=testcase.security_flag, is_crash=is_crash)
 
   issue = issue_tracker.new_issue()
+  logs.log(f"Instantiated a new issue object with ID: {issue.id}")
```
Comment (oliverchang) [inline src/appengine/libs/issue_management/issue_filer.py:337]: Please use single quotes for strings.

File: src/appengine/libs/issue_management/issue_filer.py [hunk 2]
```diff
@@ -465,6 +467,7 @@ def file_issue(testcase,
   issue.reporter = user_email
 
   recovered_exception = None
+
```
Comment (oliverchang) [inline src/appengine/libs/issue_management/issue_filer.py:470]: Can you please limit the scope of the changes and exclude whitespace changes like these?

File: configs/test/issue_trackers/config.yaml [hunk 3]
```diff
@@ -19,7 +19,102 @@
 #   - %SANITIZER%: The sanitizers associated with the testcase.
 #   - %SEVERITY%: The security severity of the testcase.
 #   - %PLATFORM%: The platform the testcase was found on.
-test-project:
+test-project-jira:
```
Comment (oliverchang) [inline configs/test/issue_trackers/config.yaml:22]: Are these changes required? 

I don't see any test changes that require this, and we typically wouldn't include them here either -- we prefer to inject these/mock this in the test setup instead.

File: src/appengine/private/components/testcase-detail/testcase-detail.html [hunk 4]
```diff
@@ -47,7 +47,8 @@
         max-width: 200px;
```
Comment (oliverchang) [inline src/appengine/private/components/testcase-detail/testcase-detail.html:47]: Are the changes in this file required?

**Review by oliverchang — CHANGES_REQUESTED** [review]
hi, thanks for your contribution, and sorry for the delay in reviewing.  

please see my comments below

File: configs/test/issue_trackers/config.yaml (hunk 3 above)
Comment (andrei-near) [reply configs/test/issue_trackers/config.yaml:22]: Added this as an example for other users. Renaming back test-project-monorail to test-project to avoid breaking any tests

File: src/appengine/libs/issue_management/issue_filer.py (hunk 1 above)
Comment (andrei-near) [reply src/appengine/libs/issue_management/issue_filer.py:337]: Got it

File: src/appengine/libs/issue_management/issue_filer.py (hunk 2 above)
Comment (andrei-near) [reply src/appengine/libs/issue_management/issue_filer.py:470]: it my fmt, removing

File: src/appengine/private/components/testcase-detail/testcase-detail.html (hunk 4 above)
Comment (andrei-near) [reply src/appengine/private/components/testcase-detail/testcase-detail.html]: formatter again, clearing

**Comment by andrei-near** [comment]
Hi @oliverchang . Thanks for review. Addressed the comments.

**Comment by andrei-near** [comment]
@oliverchang  can you review this please?
